plugins/kubernetes-backend/package.json

```diff
@@ -91,6 +91,7 @@
     "http-proxy-middleware": "^2.0.6",
     "lodash": "^4.17.21",
     "luxon": "^3.0.0",
+    "mock-fs": "^5.2.0",
```

Comment:
we avoid mock-fs, could you replace with a different solution? It's a pain to maintain for new node releases

Reply:
If you can guide me in the direction of how to do it I would more than willing to. I am using mock-fs because from what I was able to find the other mock solution for files/directories that is currently being used will not allow me to mock a specific location and instead requires that i mock a temporary location and then use that in the code. Since the ability to set Config.SERVICEACCOUNT_CA_PATH was removed i have been unable to find another solution (besides this one) to mock the needed ca.crt file/path that will be picked up by the new version of the package. I would appreciate any advice, pointers or thoughts on how to avoid mock-fs if that is what is needed.

Reply:
This has been the main sticking point for me and reason that it took this long to get everything together and working.

Reply:
Of course i just looked up `mock-fs alternatives` on google and this was the first result ...https://github.com/backstage/backstage/issues/20436 doh!

Reply:
From what I can tell of the tests, we need to be able to have a ca.crt at that location that `does not` work, and then the ability to restore it to the working version.

Reply:
It seems like we can use a mock directory using the existing `createMockDirectory` utility and the point to it using environment variables: https://github.com/kubernetes-client/javascript/blob/e055e5dfdad705a34d4b5b8f38fe7712b6e289bf/src/config.ts#L217

Reply:
@Rugvip thanks, it looks like that worked! I thought I had tried that before but ran into errors ... maybe they were unrelated to that specific change.